webapp/components/integrations/components/installed_incoming_webhooks.jsx

```diff
@@ -111,6 +111,7 @@ export default class InstalledIncomingWebhooks extends React.Component {
                     creator={this.state.users[incomingWebhook.user_id] || {}}
                     canChange={canChange}
                     team={this.props.team}
+                    channel={channel}
```

Comment:
`InstalledIncomingWebhook` will still be stateless (no changes to that filed needed), except the loading of the channel will be moved from the `InstalledIncomingWebhooks` render method to a Redux wrapper/container in an `index.js` file that will import `InstalledIncomingWebhook` and fill in any extra props not filled in by the parent component